### PR TITLE
Fix color of conversation in chat list

### PR DESCRIPTION
**Pull request by nqhhdev**
### Issue:

- Style for chat list item is wrong for read/unread

### Resolved

Screenshot_20231103_173533

### CODE REVIEW

**Comment by imGok** [comment]
hello we implemented google fonts in `themes.dart` is it still necessary with ur new linagora design PR ?

File: pubspec.yaml [hunk 1]
```diff
@@ -108,7 +108,7 @@ dependencies:
   linagora_design_flutter:
     git:
       url: git@github.com:linagora/linagora-design-flutter.git
-      ref: master
+      ref: apply-google-fonts
```
Comment (drminh2807) [inline pubspec.yaml:111]: do we need PR for that ?

Comment (nqhhdev) [reply pubspec.yaml:111]: Pls review my PR: 
- https://github.com/linagora/linagora-design-flutter/pull/30

After that I'll back it to ref master

**Comment by sherlockvn** [comment]
Please check other screens also to avoid side effects

**Comment by nqhhdev** [comment]
> Please check other screens also to avoid side effects

It only app for `DecoratorStyle` in chat list

**Review by hoangdat — APPROVED** [review]

**Pull request merged**